docs/api-reference/webgl/context-state.md

```diff
@@ -2,7 +2,13 @@
 
 luma.gl provides WebGL state management which enabling a 'stateless' and 'conflict-free' WebGL programming model. In the stateless model, settings can be passed as parameters to rendering commands rather than being set directly on the global state. And in the conflict-free model, even when setting global state, one part of the code does not need to worry about whether other parts are changing the global state.
 
-Note that to fully support the conflict-free model and detect changes done e.g. in other WebGL libraries, luma.gl needs to hook into the WebGL context to track state changes.
+The following functions are provided:
+* `getParameter` - Returns the value(s) of a GL context parameter
```

Comment:
> using the ending 's' to distinguish APIs is usually very confusing and error-prone...

I'm open to better suggestions!

I do see a need for asking for single values. But also want to offer the ability to get multiple values. All the luma.gl objects that derive from `Resource` follow the same pattern so if we change we could do it across the board.